src/screens/Settings/index.tsx

```diff
@@ -603,18 +610,27 @@ function Settings() {
               setDefaultSteamPath={setDefaultSteamPath}
             />
           )}
-          {isSyncSettings && (
-            <SyncSaves
-              savesPath={savesPath}
-              setSavesPath={setSavesPath}
-              appName={appName}
-              autoSyncSaves={autoSyncSaves}
-              setAutoSyncSaves={setAutoSyncSaves}
-              isProton={!isWin && wineVersion.type === 'proton'}
-              winePrefix={winePrefix}
-              runner={runner}
-            />
-          )}
+          {isSyncSettings &&
+            (runner === 'legendary' ? (
+              <LegendarySyncSaves
+                savesPath={savesPath}
+                setSavesPath={setSavesPath}
+                appName={appName}
+                autoSyncSaves={autoSyncSaves}
+                setAutoSyncSaves={setAutoSyncSaves}
+                isProton={!isWin && wineVersion.type === 'proton'}
+                winePrefix={winePrefix}
+                runner={runner}
```

Comment:
Passing `runner` here is a little unnecessary, I'd assume you're still doing it because you want to avoid re-writing a lot of the old IPC calls?